SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+This document descibes how the project is run through delegation of this authority, primarily to the Steering Committee and the Project Members.
+Wherever possible all descisions taken should be done by consensus, considering input from both active contributors and the wider community.
+
+## Advisory Board
+The primary functions of the advisory board are: to advise the steering committee on management of the project, find funding for the project to ensure the core aims of the project can be met, nominate a Ombudsperson, and to ensure that the governance of the project is running as intended.
+
+* **Membership Limit**: The board shall consist of at least 5 members and no more than 10 members.
+Within these limits, the board may increase or decrease the number of people serving on the board, including for the purpose of staggering the terms of members.
+
+* **Term Length**: All members shall be elected to serve a maximum of a two-year term, however the term may be extended until a successor has been elected. 
+A member may be re-elected at the end of their term.
+
+* **Staggering**: Members terms shall be staggered so that approximately half the number of members will end their terms in any given year.
+The term of office shall be considered to begin January 1 and end December 31 of the second year in office, unless the term is extended until such time as a successor has been elected.
+
+* **Vacancies**: The board may fill vacancies or may appoint new members to fill a previously unfilled board position, subject to the maximum number of members.
+The board shall fill vacancies as soon as possible.
+The board should work to ensure that the composition of the board represents a diverse cross-section of the solar physics community.
+This diversity should consider things such as research topic, career stage, as well as all other forms of diversity such as gender.
+
+* **Removal**: A board member may be removed by a two-thirds majority vote of the full board, if (a) the member is absent and unexcused from 50% or more meetings of the board in a twelve month period.
+The board chair is empowered to excuse members from attendance for a reason deemed adequate by the board chair.
+The chair shall not have the power to excuse him/herself from the board meeting attendance and in that case, the board vice chair shall excuse the chair.
+Or (b) for cause or no cause, if before any meeting of the board at which a vote on removal will be made the director in question is given electronic or written notification of the board’s intention to discuss her/his case and is given the opportunity to be heard at a meeting of the board.
+The board can remove the executive director by a two-thirds majority vote of the full board for cause or no cause.
+
+* **Meetings**:  The board shall have at least a meeting every quarter at times and places fixed by the board.
```

Comment:
can we mandate these meetings are online instead of "places fixed by the board"?

Reply:
It may be that some board meetings are not online, so ok to leave the general wording.

Reply:
I'm thinking from an accesibility point of view here - I think it would be good to mandate that there is at least an option to join any meeting online.

Reply:
In the current version this now reads:

> Notice of meetings shall provide the information necessary to participate in the meeting.

I don't think we need to be any more specific than this here, but others are welcome to comment.